build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
This is great. Could you rename this to `deploy.sh` ?

Reply:
got it